micromagneticmodel/driver.py

```diff
@@ -325,3 +343,19 @@ def _setup_working_directory(system, dirname, mode, append=True):
         workingdir = system_dir / f"{mode}-{next_number}"
         workingdir.mkdir(parents=True)
         return workingdir
+
+    @staticmethod
+    def _conversion_to_hms(time_difference):
+        total_seconds = math.ceil(time_difference.total_seconds())
+        hours, remainder = divmod(total_seconds, 3600)
+        minutes, seconds = divmod(remainder, 60)
+        return f"{hours:02}:{minutes:02}:{seconds:02}"
```

Comment:
Does `{hours:02}` cause problems if hours >= 100 (i.e. three digits)?

Reply:
The `{hours:02}` only enforces a minimum of two digits, not a maximum, so it will handle three or more digits without issues.